sdks/java/core/src/main/java/org/apache/beam/sdk/util/construction/PTransformTranslation.java

```diff
@@ -119,6 +119,9 @@ public class PTransformTranslation {
   public static final String COMBINE_PER_KEY_TRANSFORM_URN = "beam:transform:combine_per_key:v1";
   public static final String COMBINE_GLOBALLY_TRANSFORM_URN = "beam:transform:combine_globally:v1";
   public static final String RESHUFFLE_URN = "beam:transform:reshuffle:v1";
+  public static final String REDISTRIBUTE_BY_KEY_URN = "beam:transform:redistribute_by_key:v1";
```

Comment:
Should these get added as constants in the proto files?

Reply:
I'll do a separate change to perhaps migrate these constants to use the constants in the proto file.